locales/en/openbridge.json

```diff
@@ -80,11 +80,22 @@
     "deleteFilter": "Delete filter",
     "deleteProcessor": "Delete Processor?",
     "errors": {
+      "cantCreateProcessor": "Error while creating a Processor",
+      "cantCreateProcessorBecauseInstanceNotAvailable": "The Smart Events instance you are associating it with is in the deletion phase or does not exist anymore.",
+      "cantCreateProcessorOfThatType": "It is not possible to create a Processor of that type.",
       "cantDeleteBecauseNotExisting": "The Processor you are trying to delete, it does not exist anymore.",
       "cantDeleteTryLater": "It is not possible to delete this Processor. Try again later.",
+      "cantFindInstance": "The Smart Event Instance, related to the Processor you are looking for, does not exist. It might have been deleted.",
+      "cantFindParametersInfo": "The information related to the parameters section for this Processor is currently missing.",
+      "cantFindProcessor": "The Processor you are looking for does not exist. It might have been deleted.",
+      "cantUpdateProcessor": "Error while updating a Processor",
+      "cantUpdateProcessorBecauseNotReadyState": "You can update a Processor that it is only in an actionable state.",
```

Comment:
I'd be tempted to explicitly put "You can only update a Processor that is in either READY or FAILED state."